contributing_guidelines.md

```diff
@@ -0,0 +1,191 @@
+#<a name="section0"></a>Contributing guidelines
+Content:
+
+* [Introduction](#section1)
+* [Adopted conventions](#section2)
+* [Language of the main repository](#section3)
+* [Repository organization](#section4)
+* [Backlog](#section5)
+* [Contributing to the repository](#section6)
+    * [Main repository versus forked repositories](#section6.1)
+    * [Pull requests](#section6.2)
+    * [Contribution contents](#section6.3)
+    * [Commits and squashing](#section6.4)
+* [Releasing](#section7)
+* [Deployers and installed (RPMs and dockers)](#section8)
+* [Documentation](#section9)
+* [References](#section10)
+
+##<a name="section1"></a>Introduction
+This document is addressed to those development teams in charge of a so-called Cygnus agent, once its contribution has been agreed by Telefónica and the [Contribution Policy](./ContributionPolicy.txt) document has been signed.
+
+Within this document a development team will find detailed guidelines regarding how to contribute to the main Cygnus repository.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/943/commits/7a7e87176ef7b38c0d193fa949b7e468094f8274